tests/bgp/test_passive_peering.py

```diff
@@ -0,0 +1,237 @@
+'''
+
+This script is to test BGP passive peering on SONiC.
+
+'''
+
+import logging
+import pytest
+import time
+from tests.common.config_reload import config_reload
+from tests.common.helpers.constants import DEFAULT_NAMESPACE
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t2')
+]
+
+bgp_config_sleeptime = 90
+peer_password = "sonic.123"
+wrong_password = "wrong-password"
+
+
+@pytest.fixture(scope='module')
+def setup(tbinfo, nbrhosts, duthosts, rand_one_dut_hostname, request):
+    # verify neighbors are type sonic
+    if request.config.getoption("neighbor_type") != "sonic":
```

Comment:
please add a check to make sure device is running image version >= 202205.
older image does not support `show run bgp` on multi-asic devices

Reply:
@wenyiz2021 do you have any suggestions on how to do this?  I could also remove this as its more for troubleshooting than needed for the test.

Reply:
@wenyiz2021 I removed these commands.